src/web_app/chrome_app/manifest.json

```diff
@@ -5,7 +5,7 @@
   "manifest_version": 2,
   "app": {
     "background": {
-      "scripts": ["js/background.js"]
+      "scripts": ["js/apprtc.debug.js", "js/background.js"]
```

Comment:
Need to look at rest of CL first, but I think it'd be better to have a separate compile target for Chrome build with only the required bits in the background. E.g. apprtc.chrome.debug.js